unitTestSupportFunctions/getBodyNodes.m

```diff
@@ -0,0 +1,62 @@
+%% Function documentation
```

Comment:
Use my function documentation style (see other functions that we written by me). First comes the function signature, then the licensing information, then the documentation of the input/output variables which must be as detailed as possible (the user must be able to figure out from the documentation and not by reading the code what he/she needs to pass in) and then the function layout